modules/components/pages/processors/schema_registry_decode.adoc

```diff
@@ -66,36 +71,69 @@ schema_registry_decode:
+This processor decodes Protobuf messages to JSON documents For more information about the JSON mapping of Protobuf messages, see the https://developers.google.com/protocol-buffers/docs/proto3#json[Protocol Buffers Documentation^].
 
+== Metadata
+
+This processor adds the following metadata to processed messages:
+
+- `schema_id`: The ID of the schema in the schema registry associated with the message.
 
 == Fields
 
-=== `avro_raw_json`
+=== `avro.raw_unions`
+
+Whether Avro messages should be decoded into normal JSON (JSON that meets the expectations of regular internet JSON) rather than https://avro.apache.org/docs/current/specification/[Avro JSON^]. 
+
+If set to `false`, Avro messages are decoded as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodec[Avro JSON^].
+
+For example, the union schema `["null","string","Transaction"]`, where `Transaction` is a record name, would be decoded as:
+
+- A `null` as a JSON `null`
+- The string `"a"` as `{"string": "a"}`
+- A `Transaction` instance as `{"Transaction": {...}}`, where `{...}` indicates the JSON encoding of a `Transaction` instance.
+
+If set to `true`, Avro messages are decoded as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodecForStandardJSONFull[standard JSON^].
+
+For example, the same union schema `["null","string","Transaction"]` is decoded as:
+
+- A `null` as JSON `null`
+- The string `"a"` as `"a"`
+- A `Transaction` instance as `{...}`, where `{...}` indicates the JSON encoding of a `Transaction` instance.
+
+For more details on the difference between standard JSON and Avro JSON, see the https://github.com/linkedin/goavro/blob/5ec5a5ee7ec82e16e6e2b438d610e1cab2588393/union.go#L224-L249[comment in Goavro^] and the https://github.com/linkedin/goavro[underlying library used for Avro serialization^].
+
+
+*Type*: `bool`
+
+*Default*: `false`
+
+=== `avro.preserve_logical_types`
 
-Whether Avro messages should be decoded into normal JSON ("json that meets the expectations of regular internet json") rather than https://avro.apache.org/docs/current/specification/_print/#json-encoding[Avro JSON^]. If `true` the schema returned from the subject should be decoded as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodecForStandardJSONFull[standard json^] instead of as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodec[avro json^]. There is a https://github.com/linkedin/goavro/blob/5ec5a5ee7ec82e16e6e2b438d610e1cab2588393/union.go#L224-L249[comment in goavro^], the https://github.com/linkedin/goavro[underlining library used for avro serialization^], that explains in more detail the difference between the standard json and avro json.
+Whether to preserve logical types or transform them back into their primitive type.
 
+When set to `false`, logical types are transformed. For example, decimals are decoded as raw bytes and timestamps are decoded as plain integers. 
```

Comment:
Can we invert this statement? The default is `false`, so I think it makes more sense to talk about what happens if you set this to be `true`.

Reply:
LGTM!